.github/ISSUE_TEMPLATE/feature-suggestion.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature suggestion
+about: Suggest an idea for this project
+title: 'Feature suggestion: (fill in)'
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
We could instead write sections followed by comments, to stay consistent with the other templates:

```suggestion
# Benefits

<!-- Describe the benefits or value that the suggested feature would bring to the project or its users -->
```

Could you update the remaining part following this structure ?

```md
## <Section>

<!-- Short summary of what is expected here -->
```